settings.js

```diff
@@ -49,6 +49,7 @@ window.spinnakerSettings = {
   },
   checkForUpdates: true,
   debugEnabled: debugEnabled,
+  defaultExecutionResults: 10,
```

Comment:
I am making the default value 10 because in large-scale Spinnaker instances, 30 seems to be too much.

Reply:
Sorry, this is the settings.**js** file. I was referring to the typescript definition file, settings.**ts**. Including it in this file will change the default value for people running the UI locally, but not on deployed installations.

To that end, let's not change the default. It's been the default for 5+ years and nobody has attempted to change it. Changing it now might be confusing for people running the UI locally, e.g. if they're pointing to a production gate, and look at an app that has 25 running executions, they'll only see 10 locally, but 25 in production.